<launch>
    <arg name="world_frame" default="ugr/car_odom" />
    <arg name="base_link_frame" default="ugr/car_base_link" />

    
    <node pkg="slam_lap_count2" output="screen" respawn="true" type="lapc.py" name="lapc">
        <param name="world_frame" value="$(arg world_frame)" />
        <param name="base_link_frame" value="$(arg base_link_frame)" />
        
        
    </node>
</launch>